src/linker.rs

```diff
@@ -482,6 +482,8 @@ impl Linker {
     fn llvm_init(&mut self) {
         let mut args = Vec::<Cow<str>>::new();
         args.push("bpf-linker".into());
+        // starting from LLVM17 we need to disable cold callsite to avoid inlining issues
```

Comment:
Can you please change the comment to:

```
Disable cold call site detection. Many accessors in aya-ebpf return Result<T, E> where the layout is
larger than 64 bits, but the LLVM BPF target only supports up to 64 bits return values. Since the
accessors are tiny in terms of code, we avoid the issue by annotating them with #[inline(always)].
If they are classified as cold though - and they often are starting from LLVM17 - #[inline(always)]
is ignored and the BPF target fails codegen. 
```

Reply:
change applied